dangerfile.ts

```diff
@@ -0,0 +1,93 @@
+/* 
+If this is your first time editing/reading a Dangerfile, here's a summary:
+It's a JS runtime which helps you provide continuous feedback inside GitHub.
+
+You can see the docs here: http://danger.systems/js
+
+If you want to test changes to Danger locally, I'd recommend checking out an existing PR
+and then running the `danger pr` command from the terminal.
+
+For example:
+
+`npx danger pr https://github.com/facebook/react/pull/11865
+*/
+
+import { danger, markdown, message, warn } from 'danger';
+
+const BIG_PR_LIMIT = 600;
+const MAX_ALLOWED_EMPTY_CHECKBOXES = 2;
+const MIN_TICKED_CHECKBOXES_FOR_PRAISE = 6;
+const DESCRIPTION_TITLE_LENGTH = '# Description'.length;
+let errorNumber = 0;
+
+function warnAndGenerateMarkdown(warning: string, markdownStr: string): void {
+  errorNumber += 1;
+  warn(`${warning} (${errorNumber})`);
+  markdown(`> (${errorNumber}) : ${markdownStr}`);
+}
+
+async function splitBigPR() {
+  const linesCount = await danger.git.linesOfCode('**/*');
+  // exclude fixtures and auto generated files
+  const excludeLinesCount = await danger.git.linesOfCode('{**/schemaTypes.ts,fixtures/**,package*.json}');
+  const totalLinesCount = linesCount - excludeLinesCount;
+
+  if (totalLinesCount > BIG_PR_LIMIT) {
+    warnAndGenerateMarkdown(
+      ':exclamation: Big PR',
+      'Pull Request size seems relatively large. If Pull Request contains multiple changes, please split each into separate PRs which will make them easier to review.',
+    );
+  }
+}
+
+function updatePackageLock() {
+  const packageChanged = danger.git.modified_files.includes('package.json');
+  const lockfileChanged = danger.git.modified_files.includes('package-lock.json');
+
+  if (packageChanged && !lockfileChanged) {
```

Comment:
if packageChanged could you first check the content of the file to see if `dependencies` and|or `devDependencies` and|or `peerDependencies` have actually changed before creating the warning ?